src/test/java/automatedTests/ClockproductDetailed.java

```diff
@@ -0,0 +1,15 @@
+package automatedTests;
+
+import base.BaseTest;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import pages.LandingPage;
+import pages.ProductDetailedPage;
+
+public class ClockproductDetailed extends BaseTest {
+    @Test
+    public void productdetailedsucceed(){
```

Comment:
Can you improve this class and test name, I am not sure what this test is doing.